daphne_worker_test/tests/test_runner.rs

```diff
@@ -382,6 +394,9 @@ impl TestRunner {
         client: &reqwest::Client,
         agg_info: &InternalAggregateInfo,
     ) -> DapLeaderProcessTelemetry {
+        // NOTE(nakatsuka-y) The URL below is made sort of a hack-y way where we
+        // use url::Url deleting the path from the base URL when calling "join()"
+        // to our advantage to prevent us from having "version" in this URL.
```

Comment:
This is fine, but let's raise an issue to fix this later. Please convert this NOTE to a TODO and add a reference to the issue. (Here and below.)

Reply:
Done. See #80